bigtop-packages/src/rpm/flink/SPECS/flink.spec

```diff
@@ -106,29 +134,17 @@ bash $RPM_SOURCE_DIR/do-component-build
 
 sh -x %{SOURCE2} --prefix=$RPM_BUILD_ROOT --source-dir=$RPM_SOURCE_DIR --build-dir=`pwd`/build-target
 
-
-
 for service in %{flink_services}
 do
     # Install init script
     init_file=$RPM_BUILD_ROOT/%{initd_dir}/${service}
     bash %{SOURCE3} $RPM_SOURCE_DIR/${service}.svc rpm $init_file
 done
 
-
-%preun
-for service in %{flink_services}; do
-  /sbin/service ${service} status > /dev/null 2>&1
-  if [ $? -eq 0 ]; then
-    /sbin/service ${service} stop > /dev/null 2>&1
-  fi
-done
-
 %pre
 getent group flink >/dev/null || groupadd -r flink
 getent passwd flink >/dev/null || useradd -c "Flink" -s /sbin/nologin -g flink -r -d %{lib_flink} flink 2> /dev/null || :
 
-
 %post
 %{alternatives_cmd} --install %{config_flink} %{flink_name}-conf %{config_flink}.dist 30
 systemctl daemon-reload
```

Comment:
I don't think this belongs here: systemctl is only available on systemd-based Linux distros, Bigtop currently supports init.d